### ISSUE TITLE
Is this expected to be faster than fasterq-dump as well?

### ISSUE THREAD

**Issue by achillesrasquinha** [rvalieris/parallel-fastq-dump#44]

**Comment by rvalieris** [comment]
not really, the idea behind fasterq-dump is similar to parallel-fastq-dump, but it has the advantage of being implemented natively by the sratools authors, so if anything I expect fasterq-dump to be faster.

however fasterq-dump does not support all options from fastq-dump, so is some cases its still better to use parallel-fastq-dump.

see also, this benchmark and discussion https://github.com/rvalieris/parallel-fastq-dump/issues/41, made by  @Midnighter

**Comment by achillesrasquinha** [comment]
Thanks!
